st3/sublime_lib/_util/collection_util.py

```diff
@@ -0,0 +1,64 @@
+from collections.abc import Mapping, Sequence
```

Comment:
Rename to `collections.py` imo. "util" is already part of the module path.

Reply:
Done.